integration-tests/features/steps/gemini.py

```diff
@@ -161,3 +173,13 @@ def check_cves_for_epv(context, cves, p, v, e):
                 "{exp} CVEs expected, but {found} was found".format(exp=cves, found=cve_count)
             return
     raise Exception("{e}/{p}/{v} was not found".format(e=e, p=p, v=v))
+
+
+@then('I should get a valid report')
+def check_valid_report(context):
+    """Check if the stacks report is a valid one."""
+    response = context.response.json()
+    if context.list == 'list':
+        assert(isinstance(response['objects'], list))
+    else:
+        assert(isinstance(response, dict))
```

Comment:
do you plan any further tests for the report validation? If not, I can start working on them ;)

Reply:
@tisnik nothing more planned as of now.